src/System.Windows.Forms/src/System/Windows/Forms/Button.ButtonAccessibleObject.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Windows.Forms.Control;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class Button
+    {
+        [ComVisible(true)]
+        internal class ButtonAccessibleObject : ControlAccessibleObject
+        {
+            internal ButtonAccessibleObject(Button owner) : base(owner)
+            {
+            }
+
+            private Button OwningButton => Owner as Button;
+
+            internal override bool IsIAccessibleExSupported() => true;
```

Comment:
Please move overridden `IsIAccessibleExSupported` to `ControlAccessibleObject` so we do not need to override it multiple times in derived accessible objects.